### PR TITLE
fix: ensure analtyics are fired when activating upsell, various clean…

### PR DESCRIPTION
**Pull request by mcstover**
… up items

### CODE REVIEW

**Review by BoulderBrains — APPROVED** [review]

File: src/components/Checkout/AutoDepositUpsell.vue [hunk 1]
```diff
@@ -229,19 +233,32 @@ export default {
...
+		console.log(upsellEligible);
 
-		this.autoDepositUpsellExpVersion = autoDepositUpsellExp.version;
-		if (this.autoDepositUpsellExpVersion) {
-			this.$kvTrackEvent(
-				'Basket',
-				'EXP-CORE-191-Jan-2022',
-				this.autoDepositUpsellExpVersion,
-			);
+		if (upsellEligible) {
```
Comment (mcstover) [inline src/components/Checkout/AutoDepositUpsell.vue:247]: This check will prevent the exp version from being set and the exp showing for non-eligible users.

**Comment by BoulderBrains** [comment]
👍

File: src/components/Checkout/AutoDepositUpsell.vue [hunk 2]
```diff
@@ -229,19 +233,32 @@ export default {
...
 	created() {
-		// CORE-191 Checkout auto deposit upsell experiment
-		const autoDepositUpsellExp = this.apollo.readFragment({
-			id: 'Experiment:checkout_ad_upsell',
-			fragment: experimentVersionFragment,
-		}) || {};
+		const eligibilityCheck = this.apollo.readQuery({
+			query: eligibilityCheckQuery
+		});
+		console.log(eligibilityCheck);
```
Comment (eddieferrer) [inline src/components/Checkout/AutoDepositUpsell.vue:239]: Console log here and at line 245 below

**Review by eddieferrer — APPROVED** [review]
Looks good 👍

**Comment by BoulderBrains** [comment]
👍

**Pull request merged**